public/js/components/specs/NumberCard.spec.js

```diff
@@ -1,24 +1,61 @@
 import { shallowMount } from "@vue/test-utils";
 import NumberCard from "../NumberCard";
 
-describe("Number card", () => {
+describe("NumberCard", () => {
```

Comment:
Does the snapshot for `NumberCard` need to be updated to reflect this rename?

[`public/js/components/specs/__snapshots__/NumberCard.spec.js.snap`](https://github.com/noopkat/electric-io/blob/0737221d33778c09142f3cbfd0736012fe7e2126/public/js/components/specs/__snapshots__/NumberCard.spec.js.snap#L3)

Reply:
Yep, I think so. @jeffnhorner Run the following to update the snapshot accordingly:

```sh
npm test -- -u
```